.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -16,10 +16,6 @@ To check if any of the top-level crates need version increments, go to the zebra
 - [ ] Increment the crates that have new commits since the last version update
 - [ ] Increment any crates that depend on crates that have changed
 - [ ] Keep a list of the crates that haven't been incremented, to include in the PR
-- [ ] Use the `zebrad` crate version in the `zebra-network` user agent string
-      (currently the constant `USER_AGENT` in `zebra-network/src/constants.rs`)
-- [ ] Use the latest git tag in `README.md`
-- [ ] Use the latest git tag in `book/src/user/install.md`
```

Comment:
For anyone also taking a look: this was removed because it's already in `### Version Locations`